NEWS.md

```diff
@@ -1,4 +1,6 @@
 # assertthat 0.2.0.9000
+ 
+* `has_name` now rejects attempts to check multiple names in a single function call (@jameslamb, #46)
```

Comment:
Can you please put `()` after function names? See http://style.tidyverse.org/news.html#general-style

Reply:
sure. changed.